kythe/typescript/indexer.ts

```diff
@@ -383,7 +383,7 @@ class Visitor {
             // named 'default'.
             parts.push('default');
           } else {
-            this.todo(node, 'handle ExportAssignment with =');
+            parts.push('export =');
```

Comment:
Do we care about putting spaces within the signature here?  Maybe make it 'export='?

Reply:
:+1: